pom.xml

```diff
@@ -212,6 +127,13 @@
     <maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>
     <jacoco-maven-plugin.version>0.7.6.201602180812</jacoco-maven-plugin.version>
     <mockito.version>1.10.19</mockito.version>
+    
+    <final.war.name>cbioportal-${project.version}</final.war.name>
+
+	<db.test.driver>com.mysql.jdbc.Driver</db.test.driver>
+	<db.test.url>jdbc:mysql://127.0.0.1:3306/cgds_test</db.test.url>
+    <db.test.username>cbioportal</db.test.username>
+    <db.test.password>cbioportal</db.test.password>
```

Comment:
That's my plan. I'm actually happy for a properties file to be bundled inside the war file for those runtime properties, but I actually envision that as an overlay process, where a new war is made from a standard one built during a normal build. There'll then be a vanilla standard war, but users can override elements, put in logos, and so on, in a new project that uses this vanilla war as a dependency, and doesn't need to recompile everything to do it. However, for sanity, it'd be nice to allow an external properties for post-packaging configuration, just in case.

The main thing is we need to be careful to distinguish Maven (compile) config from Spring/portal.properties (runtime) config -- with the exception that when testing, we'll probably use Maven to generate files we can use for runtime testing. These are just separate from the actual deployment.